### DISCUSSION TITLE
Polymorphic many-to-many inverse relation?

### DISCUSSION THREAD

**Discussion by jhercog** [CodeDredd/pinia-orm#1889] [Q&A] [answered]
Couldn't find any information about it in the docs or on the GitHub repo. Vuex-ORM had a `morphedByMany()` attribute. Does Pinia ORM have an equivalent, or perhaps something similar?

**Comment by CodeDredd** [comment, marked as answer]
Not yet. But i can add it. 😄 
I nevered thought this would be used but i am wrong ^^.
For a cup of coffee i add this week for you 😉 .

**Reply by jhercog** [reply]
Thanks mate, your coffee is on it's way.

**Reply by CodeDredd** [reply]
Thanks a lot! ❤️ Always nice to get some extra juice to make something 😉

**Reply by CodeDredd** [reply]
@jhercog Well its done. I kept my word. You can check it out with the edge channel. Next release will still take some months.

**Reply by jhercog** [reply]
Great news, thank you! I'll give it a go as soon as  possible.

**Reply by jhercog** [reply]
@CodeDredd It works as expected, thank you very much.
